darkside-tests/src/utils.rs

```diff
@@ -384,3 +384,54 @@ impl TreeState {
         })
     }
 }
+
+pub async fn init_darksidewalletd() -> Result<(DarksideHandler, DarksideConnector), String> {
```

Comment:
it seems that this initializes dwld with a specific state for a given set of tests. I would rename this function with a more suitable name for its use